src/components/common/progress-bar.js

```diff
@@ -34,8 +36,9 @@ const StyledTrack = styled.div.attrs({
   background-color: ${props => props.trackColor || props.theme.progressBarTrackColor};
 `;
 
-const ProgressBar = ({percent, height = 4, isLoading, barColor, trackColor}) => (
-  <StyledTrack trackColor={trackColor}>
+/** @type {React.SFC<ProgressBarProps>} */
```

Comment:
It looks like SFC is now deprecated in Favor of FC: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/30364